src/Xlang/UndockedRegFreeWinRT/src/UndockedRegFreeWinRT/detours/disasm.cpp

```diff
@@ -500,7 +500,7 @@ PBYTE CDetourDis::AdjustTarget(PBYTE pbDst, PBYTE pbSrc, UINT cbOp,
                                UINT cbTargetOffset, UINT cbTargetSize)
 {
     PBYTE pbTarget = NULL;
-#if 1 // fault injection to test test code
+#if 1 // fault injection to test code
```

Comment:
I debated this for a bit. `test test code` and `test code` are both valid strings, but they mean very different things, and I can't be certain which is intended.

Reply:
Number three for engineering.